apps/passport-client/src/dispatch.ts

```diff
@@ -305,8 +295,8 @@ async function createNewUserSkipPassword(
     modal: { modalType: "none" }
   });
   const crypto = await PCDCrypto.newInstance();
-  const encryptionKey = await crypto.generateRandomKey();
-  await saveEncryptionKey(encryptionKey);
+  const encryptionKey = crypto.generateRandomKey();
+  saveEncryptionKey(encryptionKey);
```

Comment:
why remove the await, here and elsewhere?

Reply:
```public generateRandomKey(bits = 256): HexString {```
```export function saveEncryptionKey(key: string): void {```
Neither of these functions are async, so awaiting them does nothing.